MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -301,10 +283,16 @@
         "CollectibleStrayFairyWoodfall12",
         "CollectibleStrayFairyWoodfall13",
         "CollectibleStrayFairyWoodfall14",
-        "CollectibleStrayFairyWoodfall15",
-        "Woodfall Platforming"
+        "CollectibleStrayFairyWoodfall15"
+      ],
+      "ConditionalItems": [
+        [
+          "MaskDeku"
+        ],
+        [
+          "FD Difficult Jumps"
```

Comment:
I'm not familiar with this. How can Fierce Deity get to the Woodfall fairy fountain?

Reply:
FD can jump from the nearby ramp to the fountain with a somewhat precise roll jump. I want to make a video with all the FD jumps I wrote logic for, so stay tuned for that if you want to actually see this jump performed.